server/src/bin/main.rs

```diff
@@ -1,18 +1,32 @@
 // Copyright (c) 2023 MobileCoin Inc.
 
 use clap::Parser;
-use deqs_server::{Server, ServerConfig};
+use deqs_order_book::InMemoryOrderBook;
+use deqs_server::{Msg, Server, ServerConfig};
 use mc_common::logger::o;
 use mc_util_grpc::AdminServer;
-use std::{sync::Arc, thread::sleep, time::Duration};
+use postage::{broadcast, prelude::Stream};
+use std::sync::Arc;
 
-fn main() {
+/// Maximum number of messages that can be queued in the message bus.
+const MSG_BUS_QUEUE_SIZE: usize = 1000;
+
+#[tokio::main]
+async fn main() {
     let _sentry_guard = mc_common::sentry::init();
     let config = ServerConfig::parse();
     let (logger, _global_logger_guard) = mc_common::logger::create_app_logger(o!());
     mc_common::setup_panic_handler();
 
-    let mut server = Server::new(config.client_listen_uri.clone(), logger.clone());
+    let (msg_bus_tx, mut msg_bus_rx) = broadcast::channel::<Msg>(MSG_BUS_QUEUE_SIZE);
```

Comment:
main question is, why is `postage::Broadcast` preferable to `tokio::sync::Broadcast`? I started reading docu:
 
https://docs.rs/postage/latest/postage/broadcast/index.html
https://docs.rs/tokio/latest/tokio/sync/broadcast/

My assumption would be that the tokio one is very well tested and maintained, but maybe there's some feature or behavior that the other one has that you wanted?

Reply:
Good question, I can't remember if I had a reason to use it or if I just missed that tokio has that functionality. I'll try switching.

Reply:
Discussed over slack, I'll this this in a separate PR.